src/Sylius/Bundle/CoreBundle/Form/Extension/ChannelTypeExtension.php

```diff
@@ -75,6 +76,10 @@ public function buildForm(FormBuilderInterface $builder, array $options): void
                 'label' => 'sylius.form.channel.contact_email',
                 'required' => false,
             ])
+            ->add('contactPhoneNumber', TextType::class, [
```

Comment:
Could be improved to:
```suggestion
            ->add('contactPhoneNumber', TelType::class, [
```
Ref. https://symfony.com/doc/current/reference/forms/types/tel.html